pyproject.toml

```diff
@@ -60,6 +58,8 @@ pandas = [
 ]
 asynctest = "^0.13.0"
 pre-commit = {version = "^3.2.1", python = ">=3.8,<4.0"}
+plotly = "^5.15.0"
```

Comment:
I think we may need to edit the sections within the code that import these modules, and put some sort of `try/except` around them. Otherwise users will get an error when they try to run reflex.